batch2/create-batch-tables.sql

```diff
@@ -1,7 +1,11 @@
-CREATE TABLE IF NOT EXISTS `tokens` (
-  `name` VARCHAR(100) NOT NULL,
-  `token` VARCHAR(100) NOT NULL,
-  PRIMARY KEY (`name`)
+CREATE TABLE IF NOT EXISTS `globals` (
+  `instance_id` VARCHAR(100) NOT NULL,
+  `internal_token` VARCHAR(100) NOT NULL,
+  `worker_cores` BIGINT NOT NULL,
```

Comment:
Why is this BIGINT? We multiple by 1000 in the code.

Reply:
I don't think the size of this is sensitive to performance so BIGINT (8-byte integer) is easiest/safest.  Do you prefer something else?